server/boot/index.js

```diff
@@ -110,7 +110,7 @@ function setup() {
 	app.use( require( 'api' )() );
 
 	// attach the pages module
-	app.use( pages() );
+	app.use( pages.default() );
```

Comment:
I changed the pages export [here](https://github.com/Automattic/wp-calypso/pull/23187/files#diff-b2d6784517d1151d7e75f0287d0c6478R384) which made this necessary. We could get around this if we'd use module syntax in `boot/index.js` instead of `require`.